charts/rstudio-library/templates/_config.tpl

```diff
@@ -50,18 +50,20 @@
   {{- $keys | nindent 2 }}
 {{- else }}
 {{- range $parent, $child := $keys -}}
-  {{- if kindIs "map" $child }}
-    {{- if not ( kindIs "slice" $keys ) -}}
+  {{/* ini files may have multiple sections with the same name */}}
+  {{- $sections := ( (kindIs "slice" $child) | ternary $child ( list $child ))}}
```

Comment:
Supporting multiple `[cluster]` sections supports using different config files and scratch paths for launcher plugins.

If these configurations specify different values for `kubernetes-namespace` and RBAC is not configured correctly, RSW may fail to start up.